packages/lesswrong/lib/modules/fragments.js

```diff
@@ -103,7 +103,7 @@ registerFragment(`
     frontpage
     meta
     # body # We replaced this with content
-    # htmlBody # We replaced this with content
+    htmlBody # We replaced this with content
```

Comment:
Yeah, this is pretty scary. For large posts the HTML can easily be 1-2MB, and if you load 10 posts on the frontpage, that can easily add up to 10MB just to load the frontpage. So naturally I would prefer to avoid that.